### ISSUE TITLE
Test suite fails on INTFunctionTest

### ISSUE THREAD

**Issue by Jackenmen** [EPPlusSoftware/EPPlus#1113]
### EPPlus usage

Personal use

### Environment

Windows 10 22H2 (19045.3570), 

### Epplus version

6.2.10

### Spreadsheet application

None

### Description

I'm able to replicate this on both of my computers but I really don't have any idea *why* this test is failing for me since it seems to pass in CI.

Here are the logs:
```
PS C:\Users\Jakub\work\EPPlus\src> dotnet test --filter INTFunctionTest
  Determining projects to restore...
  All projects are up-to-date for restore.
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\net6.0\EPPlus.Interfaces.dll
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\net7.0\EPPlus.Interfaces.dll
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\netstandard2.0\EPPlus.Interfaces.dll
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\netstandard2.1\EPPlus.Interfaces.dll
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\net462\EPPlus.Interfaces.dll
  EPPlus.Interfaces -> C:\Users\Jakub\work\EPPlus\src\EPPlus.Interfaces\bin\Debug\net35\EPPlus.Interfaces.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\net35\EPPlus.System.Drawing.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\net7.0\EPPlus.System.Drawing.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\net462\EPPlus.System.Drawing.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\netstandard2.1\EPPlus.System.Drawing.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\netstandard2.0\EPPlus.System.Drawing.dll
  EPPlus.System.Drawing -> C:\Users\Jakub\work\EPPlus\src\EPPlus.System.Drawing\bin\Debug\net6.0\EPPlus.System.Drawing.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\net7.0\EPPlus.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\net35\EPPlus.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\net462\EPPlus.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\netstandard2.1\EPPlus.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\netstandard2.0\EPPlus.dll
  EPPlus -> C:\Users\Jakub\work\EPPlus\src\EPPlus\bin\Debug\net6.0\EPPlus.dll
  EPPlus.Test -> C:\Users\Jakub\work\EPPlus\src\EPPlusTest\bin\Debug\net481\EPPlusTest.dll
  EPPlus.Test -> C:\Users\Jakub\work\EPPlus\src\EPPlusTest\bin\Debug\net6.0\EPPlusTest.dll
Test run for C:\Users\Jakub\work\EPPlus\src\EPPlusTest\bin\Debug\net481\EPPlusTest.dll (.NETFramework,Version=v4.8.1)
Microsoft (R) Test Execution Command Line Tool Version 17.7.1 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed INTFunctionTest [247 ms]
  Error Message:
   Assert.AreEqual failed. Expected:<45217 (System.Int32)>. Actual:<#VALUE! (OfficeOpenXml.ExcelErrorValue)>.
  Stack Trace:
     at EPPlusTest.CalculationTests.INTFunctionTest() in C:\Users\Jakub\work\EPPlus\src\EPPlusTest\CalculationTests.cs:line 310


Failed!  - Failed:     1, Passed:     0, Skipped:     0, Total:     1, Duration: 374 ms - EPPlusTest.dll (net481)
Test run for C:\Users\Jakub\work\EPPlus\src\EPPlusTest\bin\Debug\net6.0\EPPlusTest.dll (.NETCoreApp,Version=v6.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.1 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed INTFunctionTest [99 ms]
  Error Message:
   Assert.AreEqual failed. Expected:<45217 (System.Int32)>. Actual:<#VALUE! (OfficeOpenXml.ExcelErrorValue)>.
  Stack Trace:
     at EPPlusTest.CalculationTests.INTFunctionTest() in C:\Users\Jakub\work\EPPlus\src\EPPlusTest\CalculationTests.cs:line 310


Failed!  - Failed:     1, Passed:     0, Skipped:     0, Total:     1, Duration: 108 ms - EPPlusTest.dll (net6.0)
```

It's the only test that fails for me. I checked each of the asserts separately and the first two are failing while the 3rd and 4th aren't.

Since those asserts/tests are related to dates, I'm guessing it could have something to do with my computer's regional settings so here's a bunch of stuff related to it from my PC:
![image](https://github.com/EPPlusSoftware/EPPlus/assets/6032823/11dadc16-a0dd-4ba7-a086-4a56744420ff)
![image](https://github.com/EPPlusSoftware/EPPlus/assets/6032823/f40667f6-57e2-45bb-afc3-6b9ef6823c4d)
```
PS C:\Users\Jakub> Get-Culture | Format-List -Property * -Expand Both

Parent                         : pl
LCID                           : 1045
KeyboardLayoutId               : 1045
Name                           : pl-PL
IetfLanguageTag                : pl-PL
DisplayName                    : Polish (Poland)
NativeName                     : polski (Polska)
EnglishName                    : Polish (Poland)
TwoLetterISOLanguageName       : pl
ThreeLetterISOLanguageName     : pol
ThreeLetterWindowsLanguageName : PLK
CompareInfo                    : CompareInfo - pl-PL
TextInfo                       : TextInfo - pl-PL
IsNeutralCulture               : False
CultureTypes                   : SpecificCultures, InstalledWin32Cultures
NumberFormat                   : System.Globalization.NumberFormatInfo
DateTimeFormat                 : System.Globalization.DateTimeFormatInfo
Calendar                       : System.Globalization.GregorianCalendar
OptionalCalendars              : {System.Globalization.GregorianCalendar}
UseUserOverride                : True
IsReadOnly                     : True
```

**Comment by JanKallman** [comment]
Thank you for reporting this.
This test does not work under Polish culture because it cannot convert the date string back to a date using this format.
We will update the test to work correctly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.2.11 and 7.0.0
